packages/teams-components/src/components/ToggleButton/ToggleButton.test.tsx

```diff
@@ -22,6 +22,28 @@ describe('ToggleButton', () => {
     ).not.toThrow();
   });
 
+  it('should not throw error for button if aria-pressed is provided', () => {
```

Comment:
please also add a test to make sure that `aria-pressed` is indeed removed from the DOM output